pkg/kudoctl/cmd/update.go

```diff
@@ -0,0 +1,109 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/cmd/install"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	updateExample = `
+		The update argument must be a name of the instance.
+
+		# Update dev-flink instance with setting parameter param with value value
+		kubectl kudo update dev-flink -p param=value
+
+		# Update dev-flink instance in namespace services with setting parameter param with value value
+		kubectl kudo update dev-flink -n services -p param=value`
+)
+
+type updateOptions struct {
+	Namespace  string
+	Parameters map[string]string
+}
+
+// defaultOptions initializes the install command options to its defaults
+var defaultUpdateOptions = &updateOptions{
+	Namespace: "default",
+}
+
+// newUpdateCmd creates the install command for the CLI
+func newUpdateCmd() *cobra.Command {
+	options := defaultUpdateOptions
+	var parameters []string
+	updateCmd := &cobra.Command{
+		Use:     "update <instance-name>",
+		Short:   "Update installed KUDO operator.",
+		Long:    `Update installed KUDO operator with new parameters.`,
```

Comment:
Should we also mention explicitly that one can not remove existing parameters this way?